### PR TITLE
Use returned DFP size when setting the height of the sticky banner

### PR DESCRIPTION
**Pull request by regiskuckaertz**
For those browsers that don't support native sticky positioning yet, the `sticky-top-banner` module kicks off. It is waiting for the ad to be rendered before setting the height of the containing `DIV`.

Recently though, it appears the value read from the CSSOM when the `onSlotRender` callback is called is not always correct, i.e. GPT has not resized the ad slot yet. So we get around that, by using the size returned by DFP instead. This also many not always be correct, but it is closer to the truth, and we have ways to deal with dynamic resizing.

For native ads, where the size is `[0, 0]`, we still read off the CSSOM though.

### CODE REVIEW

**Review by gtrufitt — APPROVED** [review]
👍

File: static/src/javascripts-legacy/projects/commercial/modules/dfp/on-slot-render.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ define([
             reportEmptyResponse(advert.id, event);
             emitRenderEvents(false);
         } else {
+            advert.size = event.size;
```
Comment (gtrufitt) [inline static/src/javascripts-legacy/projects/commercial/modules/dfp/on-slot-render.js]: Will the event definitely always have a size?

Comment (regiskuckaertz) [inline static/src/javascripts-legacy/projects/commercial/modules/dfp/on-slot-render.js]: Yes, it is documented here: https://developers.google.com/doubleclick-gpt/reference#googletageventsslotrenderendedevent

**Comment by regiskuckaertz** [comment]
[Top Banner overlapping Nav Bar](https://trello.com/c/La9k6R2P/175-top-banner-overlapping-nav-bar)

**Pull request merged**